sdk/digitaltwins/azure-digitaltwins-core/dev_requirements.txt

```diff
@@ -0,0 +1,6 @@
+-e ../../../tools/azure-devtools
+../../core/azure-core
+-e ../../identity/azure-identity
+-e ../../../tools/azure-sdk-tools
```

Comment:
Once it's in a separate directory, we will also need to add:
```
../azure-digitaltwins-nspkg
```
**Note** This line does not the `-e` flag.

Reply:
Added